client/app/bundles/shared/components/Radiobutton.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import css from './Radiobutton.scss';
+
+type Props = {
+  id?: string,
+  value?: string | number,
+};
+
+type State = {
+  value: string | number,
+  active: boolean
+};
+
+export default class Radiobutton extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { value: this.props.value || '', active: false };
+  }
+
+  onChange = (e: SyntheticEvent<HTMLInputElement>) => {
+    this.setState({ value: e.currentTarget.value });
+  };
+
+  render() {
+    const { id } = this.props;
+    return (
+      <div>
+        <form>
+          <label htmlFor={id} className={css.switch}> Option1
```

Comment:
We should be passing in props for the Option1 text since we do not always want our component to be Option1